modules/deleting-a-namespace-using-the-web-console.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/uninstalling-container-native-virtualization.adoc
+
+[id="deleting-a-namespace-using-the-web-console_{context}"]
+= Deleting a namespace using the web console
+
+.Procedure
+
+. Navigate to *Administration* -> *Namespaces*.
+
+. Locate the namespace that you want to delete in the list of namespaces.
+
+. On the far right side of the namespace listing, select *Delete Namespace* from the
+Options menu {kebab}. If you do not have permissions to delete the namespace, the *Delete Namespace*
+option is grayed out and the option is not clickable.
```

Comment:
"... option is not unavailable."
Style Guide prohibits use of "grayed out" (p.332).

Reply:
Ah, good to know -- I will fix this in the original OCP module too.